File: 30daysOFcode_Scala/day 03.scala
import java.io._
import java.math._
import java.security._
import java.text._
import java.util._
import java.util.concurrent._
import java.util.function._
import java.util.regex._
import java.util.stream._

object Solution {



    def main(args: Array[String]) {
        val stdin = scala.io.StdIn

        val n = stdin.readLine.trim.toInt

        if(n%2!=0){
        println("Weird")}
        else if(n%2==0 && n>=2 && n<=5){
        println("Not Weird")}
        else if(n%2==0 && n>=6 && n<=20){
        println("Weird")}
        else{
        println("Not Weird")}



    }
}

File: 30daysOFcode_Kotlin/day 03.kt
import java.util.Scanner

fun main() {
    val scan = Scanner(System.`in`)

    val N = scan.nextLine().trim().toInt();

    if(N%2==1){
        println("Weird");
    }
    else{
        if(N>20){
            println("Not Weird");
        }
        else 
            if(N>=6){
            println("Weird");
        }
        else{
            println("Not Weird");
        }
    }
}
